package.json

```diff
@@ -20,60 +24,114 @@
     "mocha": "mocha --require intelli-espower-loader test/test.js --full-trace --check-leaks --exit",
     "test": "nyc npm run mocha"
   },
+  "scripts": {
+    "commitlint": "commitlint",
+    "commitmsg": "commitlint -e $GIT_PARAMS",
+    "lint": "eslint --cache lib test",
+    "ci:lint:commits": "commitlint --from=${CIRCLE_BRANCH} --to=${CIRCLE_SHA1}",
+    "lint-staged": "lint-staged",
+    "release": "standard-version",
+    "release:ci": "conventional-github-releaser -p angular",
+    "release:validate": "commitlint --from=$(git describe --tags --abbrev=0) --to=$(git rev-parse HEAD)",
+    "security": "nsp check",
+    "test": "NODE_TLS_REJECT_UNAUTHORIZED=0 jest",
+    "test:watch": "jest --watch",
+    "test:coverage": "npm run test:cover && npm run test:cover:report",
+    "test:cover": "nyc jest --collectCoverageFrom='lib/**/*.js' --coverageReporters=none --coverage",
+    "test:cover:report": "nyc report --reporter=html",
+    "ci:lint": "npm run lint && npm run security",
+    "ci:test": "NODE_TLS_REJECT_UNAUTHORIZED=0 npm run test -- --runInBand",
+    "ci:coverage": "NODE_TLS_REJECT_UNAUTHORIZED=0 npm run test:coverage -- --runInBand",
+    "defaults": "webpack-defaults"
+  },
   "files": [
     "lib/",
-    "cli.js",
-    "index.js"
+    "schemas/",
+    "LICENSE",
+    "README.md"
   ],
   "peerDependencies": {
     "webpack": "^4.0.0"
   },
   "dependencies": {
     "@shellscape/koa-static": "^4.0.4",
```

Comment:
What about move this repo under `webpack-contrib`? In case you are not available for some reason (vacation, illness or something else), this will make it possible to fix bugs without forking

Reply:
if that module was not incredibly stable, I would have forked directly to contrib. but it's not a webpack-focused fork really. anything generic, for general use, that I work on I keep in my personal account. I had done due diligence first in asking them if they would consider the very small changes needed for node 6 support, but they declined. I may even fork Koa itself in the future, as the current folks managing it are doing some really weird things.

Reply:
Why not just drop Node 6 support? Active LTS for it already finished...
https://github.com/nodejs/Release#release-schedule

Reply:
Would love to. But webpack and webpack-contrib have agreed to support node versions through maintenance LTS. webpack gets _A LOT_ of donations from large companies which stay on maint LTS longer than we'd like, but that happens to be the corporate norm. maint LTS exists so slow-to-upgrade folks have a while to make the jump. I sure would prefer sticking to active LTS, that's for sure.